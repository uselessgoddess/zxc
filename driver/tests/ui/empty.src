fn foo(x: i32) -> i32 { }

extern "C" fn main(argc: isize, _: isize) -> isize {}
