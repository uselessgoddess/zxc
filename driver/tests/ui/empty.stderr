[E0228] Error: mismatch types
   ╭─[empty:1:6]
   │
 1 │ fn foo(x: i32) -> i32 { }
   │ ──────────┬────────┬─  
   │           ╰──────────── found `@unit` 
   │                    │   
   │                    ╰─── expected `i32` 
───╯
